framework/wazuh/rbac/orm.py

```diff
@@ -1237,6 +1279,44 @@ def update_role(self, role_id: int, name: str) -> Union[bool, int]:
             self.session.rollback()
             return SecurityError.ALREADY_EXIST
 
+    def migrate_data(self, manager, source: str, target: str, from_id: Optional[int] = None,
+                     to_id: Optional[int] = None) -> None:
+        """Get the roles from the `source` database filtering by IDs and insert them into the `target` database.
+
+        Parameters
+        ----------
+        manager: RBACManager
+            The manager in charge of the table data.
+        source : str
+            Path to the database to migrate data from.
+        target : str
+            Path to the database where data will be migrated to.
+        from_id : id
+            ID which the resources will be migrated from.
+        to_id : id
+            ID which the resources will be migrated to.
+        """
+        # This is to avoid an error when trying to update default users, roles, policies and rules
+        if from_id == WAZUH_USER_ID and to_id == WAZUH_WUI_USER_ID:
+            logger.warning(f"User {from_id} and {to_id } are part of the default users and can't be updated")
+            return
```

Comment:
These lines are repeated in every Manager's `migrate_data` method. Maybe it could be moved to the `DatabaseManager.migrate_data` method to avoid the iteration of every manager, displaying a single warning message. It would also involve removing the `test_<manager>_migrate_data_with_reserved_ids` tests commented below and adding the `test_databasemanager_migrate_data_with_reserved_ids` test.